### DISCUSSION TITLE
Entrega práctica

### DISCUSSION THREAD

**Discussion by pjdiaz9731** [Launch-X-Latam/MisionFrontEnd#179] [Q&A] [answered]
Hola alguien sabe hasta qué fecha tenemos tiempo de entregar la práctica?

**Comment by iam-leo** [comment, marked as answer]
Hola explorer! El MC dijo que luego se abrirá un formulario para enviar nuestras prácticas, pero hasta el momento no nos han informado nada mas que eso.
